### PR TITLE
Feature: Mentor and Volunteer Section

### PR DESCRIPTION
**Pull request by alexanderl19**
Mentor and Volunteer Stand Section

The top and bottom of the stand are seperate elements and the bottom expands as needed to accommodate additional text. The stands are placed side by side when space allows or vertically.

Safari 2023-11-22 at 01 56 49@2x

### CODE REVIEW

File: apps/site/src/app/(home)/page.tsx [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
 	// Show landing section only if still in maintenance,
 	// otherwise show the rest of the sections
-	return process.env.MAINTENANCE_MODE_HOME ? <Landing /> : <Landing />;
+	return process.env.MAINTENANCE_MODE_HOME ? (
+		<Landing />
+	) : (
+		<>
+			<Landing />
+			<MentorVolunteer />
+			<div style={{ height: 400 }}></div>
```
Comment (samderanova) [inline apps/site/src/app/(home)/page.tsx:12]: Question: What's the purpose of this `div`? Might be better to apply some padding or margin inside the `MentorVolunteer` component.

File: apps/site/src/app/(home)/sections/MentorVolunteer/MentorVolunteer.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import stand1 from "./stand-1.png";
+import stand2 from "./stand-2.png";
+
+const MentorVolunteer = () => {
+	return (
+		<div className="grid max-w-lg lg:max-w-none lg:grid-cols-[repeat(2,450px)] lg:items-stretch justify-center m-auto gap-x-8 gap-y-12 mt-24">
+			<Stand
+				imageSrc={stand1}
+				header="Apply to be a Mentor"
+				description="Interested in inspiring the next generation of developers? Our mentors are vital to making HackUCI 2023 successful and enjoyable for our hackers."
```
Comment (samderanova) [inline apps/site/src/app/(home)/sections/MentorVolunteer/MentorVolunteer.tsx:11]: Issue (here and later in the file): "HackUCI 2023" should be "IrvineHacks 2024".

File: apps/site/src/app/(home)/sections/MentorVolunteer/Stand.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	imageSrc,
+	header,
+	description,
+	buttonText,
+	buttonHref,
+}) => {
+	return (
+		<div className="max-w-[450px] text-center flex flex-col items-stretch">
+			<div className="relative aspect-[1/0.7] z-10">
+				<h2 className="absolute mt-[4.5%] text-2xl z-20 w-full text-[#FFDA7B]">
```
Comment (samderanova) [inline apps/site/src/app/(home)/sections/MentorVolunteer/Stand.tsx:25]: Suggestion: Might be worth adding the colors we use in Tailwind classes as color variables in our Tailwind config.

File: apps/site/src/assets/backgrounds/landing-background.png
Comment (samderanova) [inline apps/site/src/assets/backgrounds/landing-background.png:1]: Question: Was there a reason behind switching to a png here?

File: apps/site/src/app/(home)/sections/MentorVolunteer/MentorVolunteer.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import Stand from "./Stand";
+import stand1 from "./stand-1.png";
+import stand2 from "./stand-2.png";
+
+const MentorVolunteer = () => {
+	return (
+		<div className="grid max-w-lg lg:max-w-none lg:grid-cols-[repeat(2,450px)] lg:items-stretch justify-center m-auto gap-x-8 gap-y-12 mt-24">
```
Comment (samderanova) [inline apps/site/src/app/(home)/sections/MentorVolunteer/MentorVolunteer.tsx:7]: Issue: The stands seem to be a little too close together compared to the design.

**Review by samderanova — CHANGES_REQUESTED** [review]
Looking good so far, just a few minor issues!

File: apps/site/src/app/(home)/sections/MentorVolunteer/Stand.tsx (hunk 3 above)
Comment (alexanderl19) [reply apps/site/src/app/(home)/sections/MentorVolunteer/Stand.tsx:25]: I was actually working on designing a set of color scales that match the colors in the Figma, however that's probably outside the scope of this issue.

File: apps/site/src/assets/backgrounds/landing-background.png
Comment (alexanderl19) [reply apps/site/src/assets/backgrounds/landing-background.png:1]: To allow the water texture to show underneath the boat.

Code 2023-11-22 at 13 56 49@2x

File: apps/site/src/app/(home)/sections/MentorVolunteer/MentorVolunteer.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+			<Stand
+				imageSrc={stand1}
+				header="Apply to be a Mentor"
+				description="Interested in inspiring the next generation of developers? Our mentors are vital to making HackUCI 2023 successful and enjoyable for our hackers."
+				buttonText="Mentor Application"
+				buttonHref="google.com"
```
Comment (alexanderl19) [inline apps/site/src/app/(home)/sections/MentorVolunteer/MentorVolunteer.tsx:13]: Before I forget, do you happen to have to actual urls?

cc: @samderanova

File: apps/site/src/app/(home)/page.tsx (hunk 1 above)
Comment (alexanderl19) [reply apps/site/src/app/(home)/page.tsx:12]: Oh I was using this div to test the background earlier. It should have been removed.

File: apps/site/src/app/(home)/sections/MentorVolunteer/MentorVolunteer.tsx (hunk 5 above)
Comment (samderanova) [reply apps/site/src/app/(home)/sections/MentorVolunteer/MentorVolunteer.tsx:13]: No, not yet. I'll create a separate issue to add them once they're ready though.

**Comment by samderanova** [comment]
Might just be me, but the stands still seem to be pretty close.

image

File: apps/site/src/app/(home)/sections/MentorVolunteer/MentorVolunteer.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import Stand from "./Stand";
+import stand1 from "./stand-1.png";
+import stand2 from "./stand-2.png";
+
+const MentorVolunteer = () => {
+	return (
+		<div className="grid max-w-lg lg:max-w-none lg:grid-cols-[repeat(2,450px)] lg:items-stretch justify-center m-auto gap-y-8 gap-x-24 mt-24">
+			<Stand
+				imageSrc={stand1}
```
Comment (samderanova) [inline apps/site/src/app/(home)/sections/MentorVolunteer/MentorVolunteer.tsx:9]: Issue: On mobile views, the stands aren't centered. We should also add some additional margin to the sides.

image

**Review by samderanova — CHANGES_REQUESTED** [review]
Sorry to keep nagging you on this, but there's one more issue with the stands.

File: apps/site/src/app/(home)/sections/MentorVolunteer/MentorVolunteer.tsx (hunk 6 above)
Comment (alexanderl19) [reply apps/site/src/app/(home)/sections/MentorVolunteer/MentorVolunteer.tsx:9]: The off centered-ness is mostly baked into the asset itself. I've added an uneven padding on the stand section to counteract this and add spacing on the mobile view.

**Review by samderanova — APPROVED** [review]
Thanks for working on this! Will be squash merging

**Pull request merged**